src/util/ws_message.py

```diff
@@ -13,7 +13,23 @@
 #           }
 
 
-def format_response(incoming_msg: Dict[str, Any], response_data: Dict[str, Any]) -> str:
+Request = TypedDict(
```

Comment:
I think the Request and Response objects are basically the same, so we should unify them

Reply:
Done